services/download/src/test/resources/application.yaml

```diff
@@ -0,0 +1,29 @@
+---
+spring:
+  flyway:
+    enabled: true
+    locations: classpath:/db/migration
+  datasource:
+    url:  jdbc:tc:postgresql:11.8:///databasename?TC_TMPFS=/testtmpfs:rw
+#    if
+#    driver-class-name: org.testcontainers.jdbc.ContainerDatabaseDriver
```

Comment:
I know this comments are copied from other application.yaml files, but can we remove them from here? I guess they were there in the beginning while we were trying to use TestContainers.